lib/index.js

```diff
@@ -122,13 +147,13 @@ exports.Model = class SchwiftyModel extends Model {
         let validation = this.tryValidate(joiSchema, json);
```

Comment:
Honestly I can't find where that came from, I'm just nixing the function and doing `let validation = Joi.validate(json, joiSchema);`